webview/src/setup/components/App.tsx

```diff
@@ -128,6 +134,7 @@ export const App: React.FC = () => {
         title="Studio"
         sectionCollapsed={sectionCollapsed}
         setSectionCollapsed={setSectionCollapsed}
+        hasError={!cliCompatible}
```

Comment:
Noticed that the studio section shows the "DVC Not Available" screen if DVC is not compatible but shows the setting screen if the project is not initialized. Is this expected?

Reply:
That is how I set it up. Connecting to Studio is a secondary concern. We also can't check to see if Studio is connected if we don't have access to the CLI (need to run `dvc config studio.token`).